creating_packages/package_dev_flow.rst

```diff
@@ -0,0 +1,240 @@
++--------------------+--------------------+
+
+.. code-block:: bash
+
+    $ conan package . --source-folder=tmp/source --install-folder=tmp/install --build-folder=tmp/build --package-folder=tmp/package
+
+    PROJECT: Generating the package
+    PROJECT: Package folder C:\Users\conan\example_conan_flow\tmp\package
+    PROJECT: Calling package()
+    PROJECT package(): Copied 1 '.h' files: hello.h
+    PROJECT package(): Copied 2 '.lib' files: greet.lib, hello.lib
+    PROJECT: Package 'package' created
+
+$ conan export-pkg
+^^^^^^^^^^^^^^^^^^
+
+When you have checked that the packaged is done correctly, you can generate the package in the local cache. Note that the package is
+generated again to make sure this step is always reproducible.
+
+This parameters takes the same parameters as ``package()`` but in this case there is no ``package-folder`` as it is done in the cache.
+
++--------------------+--------------------+
+| Input folders      | Output folders     |
++====================+====================+
+| ``source-folder``  | --                 |
+|                    |                    |
+| ``install-folder`` |                    |
+|                    |                    |
+| ``build-folder``   |                    |
++--------------------+--------------------+
+
+..  code-block:: bash
+
+    $ conan export-pkg . user/testing --source-folder=tmp/source --install-folder=tmp/install --build-folder=tmp/build
+
+    Packaging to 6cc50b139b9c3d27b3e9042d5f5372d327b3a9f7
+    Hello/0.1@user/channel: Generating the package
+    Hello/0.1@user/channel: Package folder C:\Users\conan\.conan\data\Hello\0.1\user\channel\package\6cc50b139b9c3d27b3e9042d5f5372d327b3a9f7
+    Hello/0.1@user/channel: Calling package()
+    Hello/0.1@user/channel package(): Copied 2 '.lib' files: greet.lib, hello.lib
+    Hello/0.1@user/channel package(): Copied 2 '.lib' files: greet.lib, hello.lib
+    Hello/0.1@user/channel: Package '6cc50b139b9c3d27b3e9042d5f5372d327b3a9f7' created
+
+$ conan test
+^^^^^^^^^^^^
+
+The finally step to test the package for consumer is the test command. This step is quite straight-forward:
+
+.. code-block:: bash
+
+    $ conan test test_package Hello/0.1@user/channel
+
+    Hello/0.1@user/channel (test package): Installing C:\Users\conan\repos\example_conan_flow\test_package\conanfile.py
+    Requirements
+        Hello/0.1@user/channel from local
+    Packages
+        Hello/0.1@user/channel:6cc50b139b9c3d27b3e9042d5f5372d327b3a9f7
+
+    Hello/0.1@user/channel: Already installed!
+    Hello/0.1@user/channel (test package): Generator cmake created conanbuildinfo.cmake
+    Hello/0.1@user/channel (test package): Generator txt created conanbuildinfo.txt
+    Hello/0.1@user/channel (test package): Generated conaninfo.txt
+    Hello/0.1@user/channel (test package): Running build()
+    ...
+
+There is often a need to repeatedly re-run the test to check the package is well generated for consumers.
```

Comment:
Add a full cycle here as summary, without using special folders, just use the default ones, which should work.

Reply:
Also, clearly state that they can run that, maybe include the first "git clone".